app/lib/methods/helpers/normalizeMessage.ts

```diff
@@ -11,17 +14,17 @@ function normalizeAttachments(msg) {
 		if (att.ts) {
 			att.ts = moment(att.ts).toDate();
 		}
-		att = normalizeAttachments(att);
+		att = normalizeAttachments(att as TMsg);
 		return att;
 	});
 	return msg;
 }
 
-export default msg => {
+export default (msg: any): IMessage | IThreadResult | null => {
```

Comment:
In the `buildMessage` method the `msg` can be `Partial<IMessage> | IThreadResult`, in `mergeSubscriptionsRooms` the `msg` can be `ILastMessage | null | undefined`.
Why are we using any here, when we know where this function is used and what types are passed?

Reply:
As previously mentioned, Diego specified me to do this because this isn't the same as an `IMessage`, it just gets converted into one; if any of those other functions are setting a return type related to this function, then we might be doing something wrong. To be honest, I'm quite unsure.